src/lib/cfa/ntp_conf.rb

```diff
@@ -137,6 +157,8 @@ def last
       # @param [Record] record
       def <<(record)
         @augeas_tree.add(record.augeas[:key], record.augeas[:value])
+        # TODO: nasty workaround to survive multiline key with long comments
+        @augeas_tree.all_data.last[:multiline] = record.augeas[:multiline]
```

Comment:
No, multiline is not real key and I use it only to overwrite it above in save method. it is possible, but not with current implementation of record, where record is one augeas element. and these comments will be siblings. So it is a bit work-around only for autoyast.